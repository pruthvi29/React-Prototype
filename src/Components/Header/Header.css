.Header {
  font-family: "Segoe UI", Frutiger, "Frutiger Linotype", "Dejavu Sans", "Helvetica Neue", Arial, sans-serif;
}

.App-logo{
	height: 70%;
}