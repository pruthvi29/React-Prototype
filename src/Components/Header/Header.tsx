import * as React from 'react';
import './Header.css';

const logo = require('./../../Images/set_logo.png');

class Header extends React.Component<{}, {}> {
  render() {
    return (
      <div>
      <nav className="pt-navbar">
      <div className="pt-navbar-group pt-align-left">
      <img src={logo} className="App-logo" alt="logo" />
      </div>
      </nav>
      </div>
    );
  }
}

export default Header;