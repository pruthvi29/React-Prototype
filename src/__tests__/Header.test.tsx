import * as React from 'react';
import Header from '../Components/Header/Header';
  import '../Components/Header/Header.css';
const renderer = require('../../node_modules/react-test-renderer');
test('Testing Header Component', () => {
  const component = renderer.create(
    <Header></Header>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
 });