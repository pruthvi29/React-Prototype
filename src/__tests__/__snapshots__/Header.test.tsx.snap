// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Component 1`] = `
<div>
  <nav
    className="pt-navbar"
  >
    <div
      className="pt-navbar-group pt-align-left"
    >
      <img
        alt="logo"
        className="App-logo"
        src="set_logo.png"
      />
    </div>
  </nav>
</div>
`;
